apps/payments/next/app/[locale]/[offeringId]/checkout/[interval]/[cartId]/error/page.tsx

```diff
@@ -67,6 +70,13 @@ export default async function CheckoutError({
   const l10n = getApp().getL10n(locale);
   const [cart] = await Promise.all([cartPromise]);
 
+  recordEmitterEventAction(
+    'checkoutFail',
+    { ...params },
+    searchParams,
+    'stripe'
```

Comment:
Currently no easy way to determine the PaymentProvider, which is why it's hard coded to `stripe` for now.

This ticket was created to add the missing functionality. [FXA-10503](https://mozilla-hub.atlassian.net/browse/FXA-10503)

Reply:
This can be inferred via the subscription object's `collection_method`, as it's set to `send_invoice` when the payment is being handled by PayPal. Looks out of scope for this PR though, so added a comment to FXA-10503